resources/default-dump.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
default-dump.sh feels a bit too generic to me. How about `openhim-configuration-export.sh`